### ISSUE TITLE
Decals for mission-critical information should be converted to raw textures instead.

### ISSUE THREAD

**Issue by sebastiannielsen** [loiste-interactive/infra-issues#507]
In some areas, decals are used for information that are required for progressing in game, such as codes or other clues to solving puzzles.

Since the engine seem to deprioritize decals as less-important when it can't cope, I would suggest making the decals for mission-critical information to textures/materials instead. Eg, the decal + the wall texture where the decal is applied, is both made to one extra texture/material, that is then applied to a cut-out block where the decal originally was.

Note that this is a global issue that is applicable for multiple levels. One example of such convertible decal is the number on the building in hacker's den.
Another example is the B2 door sign in tunnel1, and also the D3 and D4 sign on the flood doors. (since the button panel refers to this). The other door signs can be kept as decals since those are not required to progress.

Note that such replacements are NOT recommended for decals that are not important for progressing in game, one example is the mushroom logo or the sleeping quaters decal. Its just information required for the main game story/progress and any clues/information required to progress in easter eggs that should be replaced.

**Comment by Yuccaa** [comment]
DEV note: using overlays instead of decals should do the tick without new textures.

**Comment by sebastiannielsen** [comment]
Yeah, good idea. But I think that very important information should still be made to textures so the engine don't mess with that.

So very important information (like codes and clues, example, hackerden number and D3/D4 sign, and the "2" sign on the heavy door in the beginning of forgotten world) = textures.
Less than important information, like the "B2" sign for easter egg and the pipe numbers in water treatment grit classifier = overlays.
Nonimportant information, like generic signs, like the "1" sign on the broken heavy door, and other door signs in tunnel, can be kept as decals to improve FPS and rendering.

**Comment by aleksijuvani** [comment]
The engine previously decreased the amount of decals based on the graphics settings, but I've made sure that it no longer does that. Is this still happening in some other cases?
